firmware/hw_layer/mmc_card.cpp

```diff
@@ -332,15 +332,13 @@ static BaseBlockDevice* initializeMmcBlockDevice() {
 		return nullptr;
 	}
 	
-	if (!engineConfiguration->isSdCardEnabled) {
+	if (!engineConfiguration->isSdCardEnabled || mmcSpiDevice == SPI_NONE) {
```

Comment:
Did we mean to have this mmc change in this HW PR?

Reply:
this is now in master https://github.com/rusefi/rusefi/commit/1f83047ddd72c93a09c059a4f2e5a3eaad4dc9f6